README.md

```diff
@@ -9,76 +9,92 @@ SAML: https://wiki.oasis-open.org/security/FrontPage
 ECP: http://docs.oasis-open.org/security/saml/Post2.0/saml-ecp/v2.0/saml-ecp-v2.0.html
 
 ## Setup
-To build the project, execute `mvn clean install -nsu` at the project root.
-The `distribution` module will contain a full package of the deployment after the build.
-Tests can be run with the script:
-
-`<PATH_TO_PROJECT>/distribution/command-line/target/command-line-<VERSION>-bin/bin/samltest.sh`.
-
->NOTE
->
->In order for the test kit to execute properly, you must configure both
-the test kit's and your IdP's/SP's metadata, as well as implement plugins
+To build the project, execute `gradle build` at the project root.
+The `distribution/command-line` module will contain a full package of the deployment after the build.
+
+Tests can be run with the `samlconf` script under `<PATH_TO_PROJECT>/distribution/command-line/build/distributions/samlconf-<VERSION>/bin/`.
+Build the project will create the distribution, to unzip it, execute `gradle build installDist`, which will unzip it and place it under `build/install`.
```

Comment:
✏️ Since we're already building the unzipped distro, do we really need to provide this direction? Maybe just point out that there are zip and tar archives also available after a full build?

Reply:
Agreed. I'm even confused as to what the point of running `gradle build installDist` is if you've already told me I can run the tests in the sentence above it.